modules/ipi-install-preparing-to-deploy-with-virtual-media-on-the-baremetal-network.adoc

```diff
@@ -0,0 +1,123 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+[id="preparing-to-deploy-with-virtual-media-on-the-baremetal-network_{context}"]
+= Preparing to deploy with Virtual Media on the baremetal network
+
+If the `provisioning` network is enabled and you want to expand the cluster using Virtual Media on the `baremetal` network, use the following procedure.
+
+.Prerequisites
+
+There is an existing cluster with a `baremetal` network and a `provisioning` network.
```

Comment:
@johnwilkins one more tiny thing: Could you make this a bullet? 

> A bulleted list of conditions that must be satisfied before the user starts following this module. 
https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
Sure. I just didn't put a bullet, since there wasn't a list of things.